### DISCUSSION TITLE
File .min don't generate when i build my project

### DISCUSSION THREAD

**Discussion by lolotetsoulo** [stephanrauh/ngx-extended-pdf-viewer#1749] [Q&A]
Hi,

When i use ngx-extended-pdf-viewer on test, i have no problem.

so i decide to build my project to use it in production but i got a error.

![error_min](https://user-images.githubusercontent.com/47113741/234242191-9a19e29e-4fd8-4041-aaa2-92437239efcc.png)

and when i check the build folder:
![folder_build](https://user-images.githubusercontent.com/47113741/234242639-734cd75a-bfe7-4437-b94a-e2fd4b8f495a.png)

i notice i don't have this file.

Have i forget to set something in angular.json.

Actually, i just add this:

![setting_angular](https://user-images.githubusercontent.com/47113741/234243387-6bbb3e78-f796-44f0-8a39-075a8fc95a0c.png)

Thanks for your feedback ! :)

**Comment by stephanrauh** [comment]
Your screenshot looks fine. You can have a look at the assets folder of your node_modules/ngx-extended-pdf-viewer folder. Maybe the minified files are missing there? The folder ought to look like so: https://unpkg.com/browse/ngx-extended-pdf-viewer@16.2.0/assets/

**Comment by lolotetsoulo** [comment]
it's what i have in the node_module folder.
![image](https://user-images.githubusercontent.com/47113741/234260917-8ac1d78f-6e31-4a26-9f48-750db40b0ca8.png)

i don't have a single .min file

**Reply by stephanrauh** [reply]
Delete the node_modules/ngx-extended-pdf-viewer folder and run `npm run install` again. That should do the trick.

If that doesn't work, there's a workaround: set `[minifiedJSLibraries]="false"`.

The minified files are a very compact version of the JavaScript files. They do the same, only faster.

**Reply by lolotetsoulo** [reply]
ok i will do this!

Thank you

**Reply by lolotetsoulo** [reply]
i delete the node_modules/ngx-extended-pdf-viewer folder and i run `npm install` but nothing change , i don't have minified files in the build folder

so i set the param `[minifiedJSLibrairies]="false"` and now it's work without minified files in the build folder

thanks again

**Reply by stephanrauh** [reply]
Yeah, this approach works unless performance is an issue to you. I still wonder why your *.min.js files are missing. Which version of ngx-extended-pdf-viewer are you using? Did I upload a bad version?

**Reply by lolotetsoulo** [reply]
![image](https://user-images.githubusercontent.com/47113741/234492819-f507b557-02c1-40fb-aac9-ecac4e266e41.png)

i actually use this version. This version is too old?

**Reply by stephanrauh** [reply]
There's nothing wrong with your version, unless you need XFA forms with two-way binding. TBH, your version is the most current version I've release for production use. I'm a bit unhappy with the quality of version 15 and 16, but the most recent 16.2.x version is mostly OK. I only asked because I wanted to double-check if I've released a bad version. I've automated the release process, so that's next to impossible, but you never know.

On the other hand, if you want to do me a favor, take version 17 for a spin. I've called it "alpha" because I don't have enough spare time for a thourough test. By the look of it, it'll stay an alpha version until June because I'm short of time. So I appreciate every help I can get!
